airflow/www/views.py

```diff
@@ -5727,15 +5726,6 @@ def set_task_instance_state(
         except Exception:
             flash("Failed to set state", "error")
 
-    @action("set_running", "Set state to 'running'", "", single=False)
```

Comment:
I agree with you on the removal of this feature. However, what is the case for which we have introduced this? Don't we want to mark this as deprecated before deleting this?

Reply:
Thanks for the comments. I tried to trace-back in GIT the history and the "oldest" lines in the removed code were 6+ years ago added. But in a case of UI re-factoring. Most of the other lines were re-factored multiple times since beginning. Could not find any traces in Release Notes on this. But history goes back to early 1.10 releases (1.10.2) in Jan 2019.

I believe at this time the concepts of Airflow were totally different, schedulers behaved also different in many details. I would also believe there might have been a use case in the past. But I was thinking hard and can not imagine a use case where this option is bringing any benefit (in contrast to users mis-lead by thinking they can really trigger a start manually).

I also thought multiple times about this bug report and nobody responded. So I believe this option is actually of no-use. But I am happy if somebody convinces me of a use case where this option is beneficial.

I would not deprecate it if it is of no use. If there is a real use of this function I'd rather keep it with a confirmation/warning dialog.

Reply:
I would consider this as a bug. Just setting a taskinstance as running but there being no corresponding active job for it would be the reason for it becoming a zombie.

Reply:
I believe the web UI component we don't consider as a public interface so should be ok to remove https://airflow.apache.org/docs/apache-airflow/stable/public-airflow-interface.html#what-is-not-part-of-the-public-interface-of-apache-airflow

Reply:
Yes I agree. This is a bug fix no need for any deprecation notice